source/elements/oneMKL/source/domains/rng/device_routines/device-distributions.rst

```diff
@@ -0,0 +1,95 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_distributions:
+
+Device Distributions
+====================
+
+
+oneMKL RNG routines are used to generate random numbers with different types of distribution. Each function group is 
+introduced below by the type of underlying distribution and contains a short description of its functionality, as well 
+as specifications of the call sequence and the explanation of input and output parameters. The Device Continuous 
+Distribution Generators table and Device Discrete Distribution Generators table list the random number generator routines 
```

Comment:
Am I right that `list` is a verb here?

Reply:
indeed it is. Do you suggest to remove 'the'?

Reply:
No. I'd suggest to replace the verb with any synonym, e.g. `mention`

Reply:
changed